src/scripts/components/board-exporter.js

```diff
@@ -3,7 +3,8 @@ import UserStore from '../stores/user';
 
 const Format = {
 	CSV:  'csv',
-	JSON: 'json'
+	JSON: 'json',
+	PLAINTEXT: "plaintext"
```

Comment:
Use single quotes instead of double quotes with plain strings such as these. 
Align the values on the longest element. Like so:
```
CSV:       'csv',
JSON:      'json',
PLAINTEXT: 'plaintext'
```